packages/documentation/src/stories/components/textarea/textarea.stories.ts

```diff
@@ -227,31 +183,11 @@ export const Default: Story = {};
 export const FloatingLabel: Story = {
   parameters: {
     controls: {
-      exclude: ['Hidden Label', 'Size', 'Rows', 'Helper Text', 'Disabled', 'Validation'],
+      exclude: ['Hidden Label', 'Rows', 'Helper Text', 'Disabled', 'Validation'],
     },
   },
```

Comment:
These excluded controls are legacy code, when you see them you can remove them completely.